styles.css

```diff
@@ -0,0 +1,69 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: -1px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  height: 472px;
+  top: 92px;
+  background: url(/images/header-shapes\ mobile@2x.png);
```

Comment:
Please add the background images as they appear in the figma design. Your page currently has none:
<img width="699" alt="Screenshot 2022-10-28 at 11 19 35" src="https://user-images.githubusercontent.com/48546992/198541906-40aa28c0-aa23-4b0f-a283-a480af1e35a6.png">